clone-mdb-docs-repos.py

```diff
@@ -54,66 +72,57 @@
 def run(*args):
     return subprocess.check_call(list(args))
 
-def clone_repo(user, repo):
+def fetch_and_clone_repo(gh_org, repo):
+        os.chdir(path_to_repos)
+        os.getcwd()
         print("######## cloning " + repo + " ...")
-        run("git", "clone", "git@github.com:" +  github_username + "/" + repo + ".git")
-        os.chdir(repo)
-        run("git", "remote", "add", "upstream", "git@github.com:" + user + "/" + repo + ".git")
+        run("gh", "repo", "fork", "https://github.com/" +  gh_org + "/" + repo + ".git", "--clone=true")
+        os.chdir(path_to_repos + repo)
         run("git", "fetch", "--all")
         if repo == "docs-realm-sdks":
             run("git", "branch", "-u", "upstream/latest")
         else:
             run("git", "branch", "-u", "upstream/master")
-        os.chdir("..")
+        os.chdir(path_to_repos)
 
-def clone_tengen():
-    for repo in tengen_repo_list:
+
+def fetch_and_clone_repos_by_org(gh_org, org_repo_list):
+    for repo in org_repo_list: 
         try:
             # tries to change to repo directory. if it doesn't exist, script clones the repo.
-            os.chdir(repo)
-            os.chdir("..")
+            os.chdir(path_to_repos + repo)
             not_cloned.append(repo)
             print("######## repo", repo, "exists. skipping.")
         except: 
             try:
-                clone_repo(tengen, repo)
+                fetch_and_clone_repo(gh_org, repo)
```

Comment:
if you do add a `repos_cloned` list, after this line would be a good place to append each repo to the list